Fix `value.get()` typo in `makeAddPgTableConditionPlugin` example

Tam opened the pull request:
## Description

Adds the missing `.get()` to the second `makeAddPgTableConditionPlugin` example.

## Performance impact

None

## Security impact

None

## Checklist

N/A

benjie reviewed (approved):
Thanks!

benjie commented:
@Tam Does this mean we can remove the "TODO: test this plugin works" comment above it now? (I.e. have you tested it works, or are additional changes required?)

Tam commented:
@benjie I've not tested the exact code in the example, but my tested and working version is almost identical aside from the names of tables and columns, and one comparison changed from an equals to an ilike. Happy to switch the example out with my working version, or we can leave it as is. Either way I've removed the TODO comment and added an explicit import for `TYPES`.

benjie commented on postgraphile/website/postgraphile/make-add-pg-table-condition-plugin.md:
````diff
@@ -80,8 +80,8 @@ those where a particular user has posted in (posts are stored in
 
 ```ts
 import { makeAddPgTableConditionPlugin } from "postgraphile/utils";
+import { TYPES } from '@dataplan/pg';
````
```suggestion
import { TYPES } from 'postgraphile/@dataplan/pg';
```

benjie commented:
Perfect; looks good to me - thanks!

benjie commented on postgraphile/website/postgraphile/make-add-pg-table-condition-plugin.md:
````diff
@@ -80,8 +80,8 @@ those where a particular user has posted in (posts are stored in
 
 ```ts
 import { makeAddPgTableConditionPlugin } from "postgraphile/utils";
+import { TYPES } from 'postgraphile/@dataplan/pg';
````
```suggestion
import { TYPES } from "postgraphile/@dataplan/pg";
```
